apps/synapse-portal-framework/src/components/Explore/ExploreWrapper.tsx

```diff
@@ -112,22 +101,7 @@ export default function RouteControlWrapper(props: RouteControlWrapperProps) {
           },
         }}
       >
-        {customRoutes.map(route => (
-          <Route
-            key={route.path}
-            path={`/Explore/${route.path}`}
-            exact={route.exact}
-          >
-            {route.synapseConfigArray &&
-              route.synapseConfigArray.map((synapseConfig, index) => (
-                <SynapseComponent
-                  key={index}
-                  synapseConfig={synapseConfig}
-                  searchParams={searchParams}
-                />
-              ))}
-          </Route>
-        ))}
+        <Outlet />
```

Comment:
Let react-router handle rendering the dynamic explore page content

Reply:
Very nice, thank you Outlet!